scripts/upload_test_results/upload_to_github.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python3
+import glob
+import os
+import argparse
+import shutil
+from git import Actor, Repo, Git
+import json
+import sys
+import logging
+import configparser
+import urllib.request
+from github import Github
+
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+def find_ver_json(commit_name):
```

Comment:
If the function does not find the specified commit_name, it should throw a prompt and exit.

Reply:
Yes, it does. Below is the try-except

Reply:
If you specified commit_name that doesn't in `versions.json`, the `for` loop will iterate through all the  elements in `versions.json` and return `None`.
```
        for ver_name in reversed(ver_data):
            if ver_name == commit_name:
                logging.info('Found existing version %s', ver_name)
                return ver_name
        -----> here return None 
    except Exception:
```